cob-stud-clients/README.asciidoc

```diff
@@ -35,9 +35,12 @@ $ sudo sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/g' /etc/ss
 
 ==== Gain root rights for admin user's
 ----
+$ sudo adduser --home /home-local/Admin Admin --force-badname
```

Comment:
why do you use `Admin` with capital `A`? we always use lower case names for accounts